lib/azure/armrest/template_deployment_service.rb

```diff
@@ -38,16 +38,13 @@ def get_deployment_operation(op_id, deploy_name, resource_group = configuration.
         TemplateDeploymentOperation.new(response)
       end
 
-      # Returns the json template as an object for the given deployment.
-      #
-      # If you want the plain JSON text then call .to_json on the returned object.
+      # Returns the raw json template for the given deployment as a string.
       #
       def get_template(deploy_name, resource_group = configuration.resource_group)
         validate_resource_group(resource_group)
         validate_resource(deploy_name)
         url = build_url(resource_group, deploy_name, 'exportTemplate')
-        response = JSON.parse(rest_post(url))['template']
-        DeploymentTemplate.new(response)
+        JSON.parse(rest_post(url))['template'].to_json
```

Comment:
@bzwei I thought about it, but thought it might be nice for people who do want to convert it. Or do you think that will just lead to confusion?